app/build.gradle

```diff
@@ -189,7 +199,7 @@ tasks.register('provisionEmulator', Exec) {
     doFirst {
         commandLine "${project.projectDir}/development/scripts/provision_emulator.sh",
                 "seedvault",
-                "system-images;android-33;google_apis;x86_64"
+                "system-images;android-34;google_apis;x86_64"
```

Comment:
But I'd split this into a separate commit so we can easily backport the rest to 13

Reply:
Done.